SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -984,6 +1009,8 @@ void USpatialSender::ProcessOrQueueOutgoingRPC(const FUnrealObjectRef& InTargetO
 	UFunction* Function = ClassInfo.RPCs[InPayload.Index];
 	const FRPCInfo& RPCInfo = ClassInfoManager->GetRPCInfo(TargetObject, Function);
 
+	EventTracer->TraceEvent(FEventSendRPC(TargetObject, Function));
```

Comment:
We also record this in the RPC service's `PushRPC` and `SendCrossServerRPC`, so it's gonna be duplicated.